.github/ciChecksScripts/wakeUpRunner.sh

```diff
@@ -1,24 +1,44 @@
 #!/bin/bash
 
 profile="cirunner"
-runner_vpc_id="vpc-8bdf97ec"
+runner_vpc_id1="vpc-8bdf97ec"
+runner_vpc_id2="vpc-"
 region_opt="--region us-west-2"
 
 # Get runner status
-runner=$(aws ec2 describe-instances --profile $profile --filters Name=tag:Name,Values=[jenkins] Name=network-interface.vpc-id,Values=[$runner_vpc_id] --query "Reservations[*].Instances[*][InstanceId]" ${region_opt} --output text | xargs)
+runner1=$(aws ec2 describe-instances --profile $profile --filters Name=tag:Name,Values=[jenkins] Name=network-interface.vpc-id,Values=[$runner_vpc_id1] --query "Reservations[*].Instances[*][InstanceId]" ${region_opt} --output text | xargs)
+# runner2=$(aws ec2 describe-instances --profile $profile --filters Name=tag:Name,Values=[jenkins2] Name=network-interface.vpc-id,Values=[$runner_vpc_id2] --query "Reservations[*].Instances[*][InstanceId]" ${region_opt} --output text | xargs)
```

Comment:
for runner2 you can remove ${region_opt} from script, because aws credential and config are set to us-west-1 by default, pse-runner and jenkins2 are in a same vpc